config.ini

```diff
@@ -1,3 +1,4 @@
 [thangs]
+apiUrl = https://production-api.thangs.com/
```

Comment:
I think that we can just use `[url]/api` for all API calls instead of introducing a new configuration variable, like you have for the staging configuration value.  That's what the addon does for e.g. text search

Reply:
Done